src/views/Predictions/components/Notification/PauseNotification.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { Box, Button, Text } from '@pancakeswap-libs/uikit'
+import useI18n from 'hooks/useI18n'
+import { useAppDispatch } from 'state'
+import { setHistoryPaneState } from 'state/predictions'
+import Notification from './Notification'
+
+const PauseNotification = () => {
+  const TranslateString = useI18n()
+  const dispatch = useAppDispatch()
+
+  const handleOpenHistory = () => {
+    dispatch(setHistoryPaneState(true))
```

Comment:
During the PAUSED state, after clicking through to my bet history via the `PauseNotification` comp, I couldn't see `HistoricalBet`s within the pane. Weirdly `results && results.length > 0` was true, and `finalResults` did contain my two bets. Annoyingly as I was debugging exactly what was happening, the pause state was cleared and I wasn't able to finish... so I'm not sure what, if any, data is missing. Just to flag that there might be something going on here.